test/conftest.py

```diff
@@ -159,17 +210,530 @@ def scratch_dir(config: dict[str, str]) -> str:
 
 
 @pytest.fixture(scope="session")
-def genome_refs(clients: dict[str, Any], ws_id: str) -> list[str]:
-    return make_genome_refs(clients["foft"], ws_id)
+def dummy_shock_file_handle(clients: dict[str, Any], scratch_dir: str) -> str:
+    """Upload a (tiny!) file to shock and return the resulting file.
+
+    :param clients: clients dictionary
+    :type clients: dict[str, Any]
+    :param scratch_dir: scratch directory containing the dummy file
+    :type scratch_dir: str
+    :return: shock handle for the uploaded file
+    :rtype: str
+    """
+    # "dummy.txt" lives in the test/data/ directory and is copied to the scratch dir during test set up
+    dummy_file_info = clients["dfu"].file_to_shock(
+        {"file_path": os.path.join(scratch_dir, "dummy.txt"), "make_handle": 1}
+    )
+    return dummy_file_info["handle"]
+
+
+@pytest.fixture(scope="session")
+def alignment_refs(
```

Comment:
added all the reference-generating fixtures in alphabetical order for ease of finding